python-package/lightgbm/basic.py

```diff
@@ -3413,7 +3417,12 @@ def model_from_string(self, model_str: str) -> "Booster":
         self.pandas_categorical = _load_pandas_categorical(model_str=model_str)
         return self
 
-    def model_to_string(self, num_iteration=None, start_iteration=0, importance_type='split'):
+    def model_to_string(
+        self,
+        num_iteration: Optional[int] = None,
+        start_iteration: Optional[int] = 0,
```

Comment:
```suggestion
        start_iteration: int = 0,
```

Reply:
ah yep, sorry about these! copy-pasting mistake. Thanks for catching it.